contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -386,9 +386,8 @@ contract DualAggregator is OCR2Abstract, Ownable2StepMsgSender, AggregatorV2V3In
   /// @return the aggregatorRoundId of the next round. Note: The report for this round may have been
   /// transmitted (but not yet mined) *before* requestNewRound() was even called. There is *no*
   /// guarantee of causality between the request and the report at aggregatorRoundId.
-  // TODO: i think we can remove this function entirely
```

Comment:
Can't we remove the requestNewRound() function?
I thought it would never be used.

Reply:
We should ask, I left it just in case we want to keep it.